frontend/src/metabase/dashboard/components/DashCard/DashCardVisualization.tsx

```diff
@@ -196,7 +200,6 @@ export function DashCardVisualization({
       question,
       result: mainSeries,
       isXray,
-      isEmbed,
```

Comment:
I assume this means it's only embed, but not including public sharing and the following flag take cares of that?

Reply:
Yes that flag was specific for embeds, more info here: https://github.com/metabase/metabase/pull/44206/files#r1644110712